frontend/src/app/account-activation/account-activation.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { AuthService } from '../auth.service';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-account-activation',
+  templateUrl: './account-activation.component.html',
+})
+export class AccountActivationComponent implements OnInit {
```

Comment:
Do we need `Account...` prefix here? all other components for account doesn't contain that.